INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -8,10 +8,11 @@
 import UIKit
 
 enum LoginCells: Int {
-    case logoImageCell = 0
-    case emptyCellOne
-    case emailTextFieldCell
+    case emptyCellOne = 0
```

Comment:
Enum with Integer rawValue starts from zero by default. Please remove it.

Reply:
I have just done it.